k2/csrc/fsa_equivalent.cc

```diff
@@ -199,6 +202,12 @@ bool IsRandEquivalent(const Fsa &a, const float *a_weights, const Fsa &b,
        (*(labels_difference.begin())) != kEpsilon))
     return false;
 
+  double best_distance = kDoubleNegativeInfinity;
```

Comment:
I think distance_cutoff would be a name than best_distance.  Also these are not distances, they are loglikes (or negated cost), so actually loglike_cutoff would be better.

Reply:
OK